### PR TITLE
Updated recipe for Einsteinium release version 1.2

### PR DESCRIPTION
**Pull request by Perelandra0x309**
Worked with Humdinger to create a recipe for the latest Einsteinium version.

### CODE REVIEW

File: haiku-apps/einsteinium/einsteinium-1.2.recipe [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	cd src
+	cp -af Engine/einsteinium_engine Daemon/einsteinium_daemon $binDir
+	cp -af Preferences/Einsteinium_Preferences $preferencesDir/Einsteinium
+	cp -af Launcher/Einsteinium_Launcher $appsDir/Einsteinium/Einsteinium\ Launcher
+
+	# create the Subscriber Example Kit
+	cp -af Engine/SubscriberExample/makefile Engine/SubscriberExample/subscriber_example \
+Engine/SubscriberExample/subscriber_example.rsrc Engine/SubscriberExample/SubscriberExample.cpp \
+Engine/EngineSubscriber.cpp Engine/EngineSubscriber.h $appsDir/Einsteinium/EngineSubscriberKit
+
```
Comment (humdingerb) [inline haiku-apps/einsteinium/einsteinium-1.2.recipe]: Would be nicer if those lines were indented twice. Possibly also only have one file per line, as with the paths they get pretty long. Or copy first the two files from Engine/ and the cd into Engine/SubscriberExample. Or cp Engine/SubscriberExample/* .

File: haiku-apps/einsteinium/einsteinium-1.2.recipe [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	# deskbar links
+	addPreferencesDeskbarSymlink $preferencesDir/Einsteinium
+	addAppDeskbarSymlink $appsDir/Einsteinium/Einsteinium\ Launcher
+
```
Comment (humdingerb) [inline haiku-apps/einsteinium/einsteinium-1.2.recipe]: remove that blank line.

**Comment by Perelandra0x309** [comment]
I think (hope) I pushed the recommended changes to my fork.

**Comment by humdingerb** [comment]
It does have a merge-commit in it, but I think GitHub's "Squash & merge" feature will fix that.

File: haiku-apps/einsteinium/einsteinium-1.2.recipe [hunk 3]
```diff
@@ -0,0 +1,97 @@
+SUMMARY="Monitor applications and system services"
+DESCRIPTION="Einsteinium provides smarter monitoring of applications and \
+system services for Haiku. It can restart applications and system services \
+that were quit or crashed, gather statistics on application usage and \
+provides customizable ranked lists of applications."
+HOMEPAGE="https://sourceforge.net/projects/esforhaiku/"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/esforhaiku/files/Einsteinium-${portVersion}_source.zip/download"
```
Comment (fbrosson) [inline haiku-apps/einsteinium/einsteinium-1.2.recipe]: You might wish to use
```
SOURCE_URI="https://downloads.sourceforge.net/esforhaiku/Einsteinium-${portVersion}_source.zip"
```

Comment (Perelandra0x309) [inline haiku-apps/einsteinium/einsteinium-1.2.recipe]: Pushed new download uri.

**Pull request merged**